src/profile-logic/gecko-profile-versioning.js

```diff
@@ -1026,5 +1026,69 @@ const _upgraders = {
     }
     convertToVersion20Recursive(profile);
   },
+  [21]: profile => {
+    // Migrate DOMEvent markers to Markers 2.0
```

Comment:
Oh, since we add the upgrader in the following commit, that means that we are changing the gecko side.